src/NServiceBus.Core.Tests/AssemblyScanner/When_checking_image_type.cs

```diff
@@ -1,29 +0,0 @@
-﻿namespace NServiceBus.Core.Tests.AssemblyScanner
-{
-    using System.IO;
-    using NUnit.Framework;
-
-    [TestFixture]
-    public class When_checking_image_type
```

Comment:
seems like you removed all related code to image detection, but I don't see this specifically tested/verified in the new validation code, can you share some more background on that change?